src/mockData/mockData.ts

```diff
@@ -48,7 +46,7 @@ const facilities: Array<Facility> = [
                     }
                 ],
                 degrees: [<Degree>'DOCTOR_OF_MEDICINE'],
-                spokenLanguages: [<Language>'JAPANESE', <Language>'ENGLISH'],
+                spokenLanguages: [],
```

Comment:
Since we're moving away from mockData to a real database, I didn't bother updating these